src/itinerary/index.tsx

```diff
@@ -76,6 +76,10 @@ const StyledItinerary = styled(Itinerary)`
     font-size: ${font.s.size};
   }
 
+  & .kirk-itineraryLocation-chevron {
+    align-self: ${props => (props.segments ? 'baseline' : 'center')};
```

Comment:
Approved by Jonas. This is because right now it's not possible to add a new item in the Itinerary list without being a place. The hack is to have the "Change of vehicle"  inside the place where the connection occurs.

Reply:
Could you comment the code then, it will be easier to understand the "why" behind it without reading this PR